pkg/Microsoft.NETCore.Platforms/runtime.json

```diff
@@ -922,18 +915,11 @@
             "#import": [ "fedora-corert", "linux-x64-corert", "fedora-x64" ]
         },
 
-        "fedora.23-corert": {
-            "#import": [ "fedora-corert", "fedora.23" ]
-        },
-        "fedora.23-x64-corert": {
-            "#import": [ "fedora.23-corert", "fedora-x64-corert", "fedora.23-x64" ]
-        },
-
         "fedora.24-corert": {
```

Comment:
I don't think we should remove these from the RID graph, as there might be others that still require this in the graph. I would keep them.